terraform/modules/stack/main.tf

```
@@ -282,6 +282,41 @@ module "bag_tagger" {
   service_discovery_namespace_id = local.service_discovery_namespace_id
 }
 
+# bag retagger - for adding checksum tags
+
+module "bag_retagger" {
+  source = "../service/worker"
+
+  service_name    = "${var.namespace}-bag-retagger-temp"
+  container_image = "975596993436.dkr.ecr.eu-west-1.amazonaws.com/uk.ac.wellcome/bag_tagger:retagger"
+
+  environment = {
+    queue_url         = "https://sqs.eu-west-1.amazonaws.com/975596993436/storage_staging_bag_retagger_input"
```

```suggestion
    queue_url         = module.bag_retagger_input_queue.url
```

otherwise the prod service is going to try to read from the staging queue

---

some weirdness with applying this terraform after the AWS outage left the service in a weird state where the env vars weren't updating - hence this